### PR TITLE
Server scaffolding

### PR DESCRIPTION
**Pull request by zship**

### CODE REVIEW

File: server/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+    - For Firefox, navigate to `about:preferences#privacy`, scroll down to the
+      "Certificates" section, and click "View Certificates". In the
+      "Authorities" tab, click "Import", then choose this project's
+      ".keys/ca-cert.pem".
+
+## Run
+
+```
+$ yarn run start
```
Comment (kimlisa) [inline server/README.md:39]: is this command supposed to work? (didn't work for me)

**Review by kimlisa — APPROVED** [review]

File: server/README.md (hunk 1 above)
Comment (zship) [reply server/README.md:39]: It sure is... hmm. Thanks for the heads-up, checking...

Comment (zship) [reply server/README.md:39]: Hm, tried a few things (starting from scratch and a few different node versions) but unfortunately haven't been able to repro. Could you send me the error you're getting when you get a chance?

Comment (kimlisa) [reply server/README.md:39]:
```
scripts/start-server: 4: .: .env: not found
```

**Review by alex-kovoy — APPROVED** [review]

File: server/README.md (hunk 1 above)
Comment (zship) [reply server/README.md:39]: I'm pretty sure it'll work if you run `make` first. That's supposed to generate the ".env" file. Fingers crossed!

**Pull request merged**